Gems/GraphCanvas/Code/Source/Components/Nodes/General/GeneralSlotLayoutComponent.cpp

```diff
@@ -190,16 +190,14 @@ namespace GraphCanvas
         ConnectionType connectionType = ConnectionType::CT_Invalid;
         SlotRequestBus::EventResult(connectionType, slotId, &SlotRequests::GetConnectionType);                
         
-        int layoutOrder = 0;
-
         SlotLayoutInfo slotInfo(slotId);
 
         if (connectionType == CT_Input)
         {
             SlotUINotificationBus::MultiHandler::BusConnect(slotId);
 
             m_inputSlotSet.insert(slotId);
-            layoutOrder = LayoutSlot(m_inputs, m_inputSlots, slotInfo);
+LayoutSlot(m_inputs, m_inputSlots, slotInfo);
```

Comment:
Whitespace is off here.
Actually every line where a variable is no longer being assigned has deleted all the previous whitespace on that line.